pkg/orchestration/wiring/testdata/ec2compute.basicdep.bundle.yaml

```diff
@@ -63,22 +63,10 @@ spec:
           secretName: compute--basic
   - name: compute--secret
     references:
```

Comment:
do you want to add another queue to the state yaml, which _will_ have dead letter environment variables?

Reply:
or create a separate test state

Reply:
The E2E tests do test this already so it's good.